styles/buttons.scss

```diff
@@ -58,6 +58,24 @@
   }
 }
 
+// help icon button
+.btn-help {
+  @include layout;
+  background: var(--button-tertiary-background);
```

Comment:
support secondaryBackground color here. (we use a `light` prop on staking dashboard).